convene-web/bin/run

```diff
@@ -10,7 +10,4 @@
 # visited in browser without doing port forwarding.
 : ${BINDING:=localhost}
 
-echo "******************************************************"
-echo "*** Starting Rails server at http://$BINDING:$PORT ***"
-echo "******************************************************"
```

Comment:
The point of this particular message was so that it was easier for folks who are getting started to know where they should point their browser to get started; does it get swallowed by overmind?